docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -100,128 +99,226 @@ In the case of errors, include human-readable error messages in the response pay
+```text title="Example connection option" lineNumbers 
+- Label: Packaging Method
+- Required: false
+- Type: Multi-select
+- Values: Extra primary packaging, Extra secondary packaging, Extra tertiary packaging
+```
+
+<!-- 
+type: tab
+title: Password 
+--> 
+
+```text title="Example connection option" lineNumbers 
+- Label: Password
+- Required: true
+- Type: password
+```
+
+<!-- type: tab-end -->
+
+
+## Sign up
+After you finish developing the app, you can sign up to be a shipping provider. Submit your app to BigCommerce and BigCommerce will register your app as a shipping provider. 
+
+### What should you provide BigCommerce?
+Send an email to
+<a href="mailto:shippingproviderapi@bigcommerce.com">shippingproviderapi@bigcommerce.com</a>.
+
+Include the following information:
+
+- Name of app
+- [Your app ID](#your-app-id)
+- Your email
+- A description of the app. The description is displayed to merchants in the store control panel.
+- Logo: A 70x70 pixel logo that represents the shipping carrier app
+- [Your service URLs](#your-service-urls) 
+- [Whether you prefer single-carrier or multi-carrier status](#single-carrier-versus-multi-carrier-shipping-providers)
+- [Configuration fields](#configuration-fields) (optional): For a list of items you need to provide, see [types of configuration options](#what-are-the-types-of-configuration-options).
+- Countries Available (optional): A list of countries where merchants can use your carrier. Merchants who have a store origin address outside this list will not be able to use your carrier. 
+  
+  In most cases, this list should be as broad as possible. For example, if your carrier operates worldwide, make it available worldwide. You can limit the countries further than what your shipping carrier offers. If you don't provide the countries available, your carrier is available worldwide (i.e. for every shipping origin). 
 
-```http title="Example carrier connection request with connection settings" lineNumbers
-POST https://example.com/shipping/carrier/connection
+After submitting your app, you will receive the a carrier ID. Both single-carrier and multi-carrier shipping providers receive one `carrier_id`.
+
+### What should you document for API users?
+ 
+We recommend that you document your carrier ID and configuration option codes for API users who wish to use your carrier. For example, API users will specify the `carrier_id` and `code` for connections options when they [connect your carrier to their store](#how-your-app-will-be-connected-to-a-store). API users specify the `code` for settings options when they define your carrier's shipping method for their shipping zones.  
+
+## What's next?

```

Comment:
the subheadings here are awkward, cuz they're kind of questions, but not really. think about other options.

Reply:
changed headings to non-questions